src/sentry/api/serializers/base.py

```diff
@@ -62,5 +62,10 @@ def __call__(self, obj, attrs, user, **kwargs):
     def get_attrs(self, item_list, user, **kwargs):
         return {}
 
+    def serialize_all(self, objects, attrs, user, **kwargs):
```

Comment:
I don't think we need to introduce a new concept to `serializers`. This bulk logic could probably live in `get_attrs`?

Reply:
broadly agree, though I think if we designed this API today we'd probably have made it bulk-first - id say if the code is messy _without_ this then lets add it, if not, lets avoid extending the API

aside if we do add it, maybe call it serialize_many as that's a common naming paradigm in similar libraries

Reply:
I don't think I fully grokked what `get_attrs` was doing when I first wrote this, but you're totally right. I'll do it this way.